src/docfx/Errors.cs

```diff
@@ -392,6 +392,9 @@ public static Error AtXrefNotFound(SourceInfo<string> source)
             public static Error XrefNotFound(SourceInfo<string> source)
                 => new Error(ErrorLevel.Warning, "xref-not-found", $"Cross reference not found: '{source}'.", source);
 
+            public static Error InvalidXrefType(SourceInfo<string> xref, string expectedXrefType, string? realXrefType)
```

Comment:
make the method name same as the error code.

Reply:
OK.